### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by eleanorakh**
Add issue templates for bug reports and feature suggestions. Thanks here to the GitHub Wizard 😂

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+ contact_links:
```
Comment (mmolina) [inline .github/ISSUE_TEMPLATE/config.yml:2]: I think there is an extra space before `contact_links:`. It looks like every line below is shifted one extra space right, and I fear that'll invalidate the YAML file

**Review by mmolina — COMMENTED** [review]
I think the `config.yml` file is not valid YAML as it is because of an extra space at the beginning of a line. Unsure if there are any side effects 🤷‍♂️

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (eleanorakh) [reply .github/ISSUE_TEMPLATE/config.yml]: Good catch! Fixed.

**Review by mmolina — APPROVED** [review]
Looks good to me! 😃

**Pull request merged**